book/3-classification.tex

```diff
@@ -400,7 +400,20 @@ \subsection{F1-score}
 
 % formula
 \begin{center}
-    [FORMULA GOES HERE]
+    \tikz
+        \node[inner sep=2pt, font=\Large] (a) {
+            {
+                $\displaystyle
+                F1 = \frac{2}{\textcolor{white!25!blue}{P}^{-1} + \textcolor{violet}{R}^{-1}} = \frac{2\textcolor{white!25!red}{TP}}{2\textcolor{white!25!red}{TP} + \textcolor{teal!60!green}{FP} + \textcolor{gray}{FN}}
+                $
+            }
+        };
+   \draw[-latex, white!25!blue, semithick] ($(a.south)+(-2.7,0.1)$) to[bend left=25] node[pos=1, left] {\scriptsize \textbf{\color{white!25!blue} precision}} +(-.6, -.5);
```

Comment:
the \scriptsize is not standard in the book formulas. can you remove it

Reply:
the colors here are not the standarized formula colors. please use the following:

% formula colors
\definecolor{nmlpurple}{HTML}{3B0280}
\definecolor{nmlcyan}{HTML}{00C8E5}
\definecolor{nmlred}{HTML}{DD4040}
\definecolor{nmlgreen}{HTML}{4EB046}
\definecolor{nmlyellow}{HTML}{E1BC29}